### PR TITLE
Modify example application

### PR DESCRIPTION
**Pull request by tomerlevi444**

### CODE REVIEW

**Comment by tomerlevi444** [comment]
Note: I've added a small change in KODContext spec as well.

**Comment by yuval-netanel** [comment]
Approved!

File: Example/KlarnaOnDemandExample/MainViewController.m [hunk 1]
```diff
@@ -15,6 +15,7 @@ - (void)viewDidLoad {
   [super viewDidLoad];
 
   [self initializeUIElements];
+  [self updateUIElements];
```
Comment (grozen) [inline Example/KlarnaOnDemandExample/MainViewController.m]: Do you need to both initialize **and** update them? How did this work before?

**Comment by grozen** [comment]
A-pproved.

File: Example/KlarnaOnDemandExample/MainViewController.m (hunk 1 above)
Comment (tomerlevi444) [inline Example/KlarnaOnDemandExample/MainViewController.m]: I called initializeUIElements on viewDidLoad, and when I received 'userReady' event, I just set registrationButton.hidden=true and preferencesButton.hidden=false.

**Pull request merged**